docs/collection-interfaces.md

```diff
@@ -44,6 +44,15 @@ This method is defined to be a fluent version of `ArrayIterator::append`. To do
 $collection->add($item1)->add($item2);
 ```
 
+### chunk
+
+```php
+/** @return self[]|static[] */
+public function chunk(int $size): array
+```
+
+This method [mirrors the behavior of `array_chunk`](https://www.php.net/manual/function.array-chunk.php) and returns a zero indexed numeric array of the current Collection based on the chunk size provided.
```

Comment:
of the current collection <- This is english, we don't need to capitalize the nouns (we can but here it doesn't make sense)

Reply:
It was meant as a proper noun, which in english is capitalized, however you are right: in the context of this README we can strive for consistency.